### PR TITLE
PER-911 - AWS TPCC PGD 2 Nodes

### PR DESCRIPTION
**Pull request by dougortiz**
Benchmark: AWS_TPCC_PGD. Issues fixed:

- TPAExec Install
- Async and Poll in benchmark pipeline
- Terraform PlugIn Cache
- Terraform Path in Workers

### CODE REVIEW

**Comment by bryan-bar** [comment]
http://localhost:8010/#/builders/8/builds/4

File: aws-tpcc-pgd/deploy/playbook-setup-tpa.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        path: /etc/fstab
+        line: "/dev/{{ item.vgname }}/{{ item.lvname }} {{ item.mountpoint }} xfs noatime 0 0"
+      loop:
+        - lvname: LVPGDATA
+          vgname: VGPGDATA
+          mountpoint: "{{ pg_data | dirname }}"
+        - lvname: LVPGWAL
+          vgname: VGPGWAL
+          mountpoint: "{{ pg_wal | dirname }}"
+      become: true
```
Comment (bryan-bar) [inline aws-tpcc-pgd/deploy/playbook-setup-tpa.yml:94]: Please remove this file.
```suggestion
```

I do not see this playbook called in `run.sh` and appears to be a duplicate of `playbook-setup-fs.yml`

https://github.com/EnterpriseDB/edb-benchmarks/blob/0251657f7ba89bdfc8f2ce2561ab7ad251759089/aws-tpcc-pgd/deploy/run.sh#L29-L35

**Review by bryan-bar — CHANGES_REQUESTED** [review]
Build completed.
Thank you for the async fix and validation step.

Please remove the unneeded file and this should be good to go.

File: aws-tpcc-pgd/deploy/playbook-setup-tpa.yml (hunk 1 above)
Comment (dougortiz) [reply aws-tpcc-pgd/deploy/playbook-setup-tpa.yml:94]: Can I remove this in the next PR?

Comment (dougortiz) [reply aws-tpcc-pgd/deploy/playbook-setup-tpa.yml]: Have already removed it in the PR for AWS TPCC REFDATA

**Pull request merged**